lib/modules/accountSwitcher.js

```diff
@@ -250,6 +250,9 @@ async function populateUser(container) {
 
 			$contents.append($gilded);
 		}
+
+		// Append link to profile page
+		$contents.append($('<a>', { href: `/u/${username}`, class: 'res-icon linkIcon' }).click(e => e.stopPropagation()));
```

Comment:
and swap `.click()` for `.on('click', e => ...)` just to clarify "this is an event listener"?